vrecord

```diff
@@ -1138,8 +1138,8 @@ export MAIN_DIALOG='
     <action signal="leave-notify-event" type="refresh">VRECORD_FORM_VALIDATION</action>
     <action signal="show" type="closewindow">HOME_DIALOG</action>
     <action signal="show" type="refresh">VRECORD_OUTPUT_NAME</action>
-    <action signal="show" condition="command_is_true( [[ \"$AUDIO_DEV_CHOICE\" != *\"Blackmagic\"* ]] && echo true)">enable:AUDIO_MODE_SR_CHOICE</action>
-    <action signal="show" condition="command_is_true( [[ \"$AUDIO_DEV_CHOICE\" == *\"Blackmagic\"* ]] && echo true)">disable:AUDIO_MODE_SR_CHOICE</action>
+    <action signal="show" condition="command_is_true( [ \"$AUDIO_DEV_CHOICE\" != *\"Blackmagic\"* ] && echo true)">enable:AUDIO_MODE_SR_CHOICE</action>
+    <action signal="show" condition="command_is_true( [ \"$AUDIO_DEV_CHOICE\" = *\"Blackmagic\"* ] && echo true)">disable:AUDIO_MODE_SR_CHOICE</action>
```

Comment:
This isn't working on my side. How about:

`[ \"${AUDIO_DEV_CHOICE/Blackmagic}\" != \"${AUDIO_DEV_CHOICE}\" ]`

Reply:
@dericed On which macOS version?

Reply:
@retokromer 10.15.7, but this is run within the gtkdialog subshell

Reply:
@dericed Thank you! Sorry, I forgot that all versions of macOS I have here at home aren’t set up with the Bash version kindly provided by Apple™…